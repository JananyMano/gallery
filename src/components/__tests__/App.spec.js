import React from "react";
import { create } from "react-test-renderer";
import App from "../App";

describe("App", () => {
  test("App component matches the snapshot", () => {
    const app = create(<App />);
    expect(app.toJSON()).toMatchSnapshot();
  });
});