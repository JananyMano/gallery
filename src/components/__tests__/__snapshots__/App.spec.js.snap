// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App component matches the snapshot 1`] = `
<div
  className="ui container"
  style={
    Object {
      "marginTop": "10px",
    }
  }
>
  <div
    className="ui segment"
  >
    <form
      className="ui form"
      onSubmit={[Function]}
    >
      <div
        className="field"
      >
        <label>
          Image Search
        </label>
        <input
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
    </form>
  </div>
  <div
    className="image-list"
  />
</div>
`;
